manifest.json

```diff
@@ -1,8 +1,14 @@
 {
+  "version": "0.1.0",
   "domain": "bchydro",
   "name": "BC Hydro",
-  "documentation": "https://github.com/emcniece/hass-bchydro",
+  "documentation": "https://github.com/sascha432/hass-bchydro",
   "dependencies": [],
   "codeowners": [],
-  "requirements": []
+  "requirements": [
+    "flake8==4.0.1",
+    "black==22.1.0"
```

Comment:
`flake8` and `black` are dev requirements, not deploy requirements - this script doesn't need them in order to run normally. What does adding them here do? I can't recall if it's the `dependencies` or `requirements` that causes Home Assistant to download the packages when this integration gets installed. 

If Home Assistant _does_ install these requirements, I'd lean toward omitting them from the manifest. Are they needed here to support the new manifest?

Reply:
In this case, there is no requirements. Home assistant installs any missing requirements during startup.

Here is the updated file

`	{
  "version": "0.1.0",
  "domain": "bchydro",
  "name": "BC Hydro",
  "documentation": "https://github.com/emcniece/hass-bchydro",
  "dependencies": [],
  "codeowners": [],
  "requirements": [],
  "iot_class": "cloud_polling",
  "config_flow": false
}`